mentor-mind-app/src/pages/SettingsPage.tsx

```diff
@@ -0,0 +1,20 @@
+import {FormLabel, Stack, TextField} from "@mui/material";
+import PinnedContainer from "../components/containers/PinnedContainer";
+import RegisterComponent from "../components/login-register/RegisterComponent";
+import LogInComponent from "../components/login-register/LogInComponent";
+import SettingsMainComponent from "../components/settings/SettingsMain";
+import SettingsPassword from "../components/settings/SettingsPassword";
+
+const SettingsPage = () => {
+    return (
+        <Stack direction={"column"} justifyContent={"space-evenly"} minHeight={"calc( 120% - 100px );"} height={"calc( 120% - 100px );"} alignItems={"center"} margin={"auto"} sx={{pb: 20, pt: 10}}>
```

Comment:
If you want some space between the forms, you can add a `spacing={8}` (or other value) property here